### ISSUE TITLE
Is there a way to just highlight and leave the syntax colors "as is" ?

### ISSUE THREAD

**Issue by yrcmpbll** [mnespor/vscode-color-identifiers-mode#16]
Hi,

first of all, I would like to thank the developers for this very useful add on, specially for people with attention disabilities, as it a a must while programming.

That said, I would like to know if there is a reliable way to avoid the "language keywords less prominent." in "These themes make variables more prominent and language keywords less prominent."

Now, if you set any theme on, it will make language keywords less prominent, permanently. The only way I find it for just "make variables more prominent" is to uninstall and install the add-on again.

In summary, I wish to have an option of just make variables more prominent by coloring them, while everything else stays the same. By now, the only way of achieving this is by a fresh installation and do not touching the "themes".

Thank you very much for your support and contribution for the dev community.

**Comment by mnespor** [comment]
Thanks for the message and the kind words!

You can use your favorite theme with the extension any time through Settings -> Theme -> Color Theme.

The extension is compatible with colorful themes (the two all-gray themes are only there for convenience). The theme picks the colors for syntax, and the extension picks the colors for variables.
